config/manager/manager.yaml

```diff
@@ -41,9 +41,7 @@ spec:
         - name: CERT_POLICY_CONTROLLER_IMAGE
           value: quay.io/stolostron/cert-policy-controller:latest
         - name: CONFIG_POLICY_CONTROLLER_IMAGE
-          value: quay.io/stolostron/config-policy-controller:latest
-        - name: KUBE_RBAC_PROXY_IMAGE
-          value: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.10
+          value: quay.io/open-cluster-management/config-policy-controller:latest
```

Comment:
This will need to be updated to `quay.io/stolostron` (That's what the CI is complaining about also)